
std_msgs/Header header

## 任务类型 1定位 2本机追踪 3TBD
int32 track_mission_type

## 任务状态 1
int32 track_state

## 目标位置
float32 pos_x
float32 pos_y
float32 pos_z
float32 pos_yaw


